cache/README.md

```diff
@@ -104,3 +106,11 @@ terraform apply terraform.plan
 7. Check the data is being sent to GA (either `someToggleId` or `!someToggleId`). You should initially be able to see the toggles dataLayer variable being set (`DLV - Toggles`) in GTM, then this data should get sent to GA as a custom dimension (note you might not be able to see this until the next day)
 8. Update [`locals.tf`](https://github.com/wellcomecollection/wellcomecollection.org/blob/main/cache/locals.tf) with values from previous terraform and re-run the terraform steps above to get the changes in to production
 9. Use the toggle to conditionally serve different UI in the same way as you would for feature flags
+
+#### Removing the test
+1. Remove the test objects from `toggler.ts` and `toggles.ts` that were added above and remove any code that responds to the toggle cookie in the same PR.
+2. Deploy the new lambdas and apply the terraform changes as per steps 2–5 above.
+3. Make a separate PR to update the [`locals.tf`](https://github.com/wellcomecollection/wellcomecollection.org/blob/main/cache/locals.tf) from the result of the previous step
+4. Wait until both PRs are approved
+5. Apply the terraform change resulting from the updated locals.tf (check the toggle cookie is no longer set in production)
+6. Merge and deploy both PRs
```

Comment:
Have we every figured out why they were separate, is it relevant in this case if they're deployed at the same time? 
(we can figure it out together bc I really don't know either)

Reply:
I'm not sure. I suppose the _deploying_ of the locals.tf update is not relevant because all that matters is that it's in `main` (for anybody else's terraform changes not to overwrite it)?

Reply:
I don't think I follow. Would you be willing to write the steps as you see them making sense?